packages/flutter/lib/src/material/app.dart

```diff
@@ -782,9 +789,14 @@ class MaterialScrollBehavior extends ScrollBehavior {
   Widget buildOverscrollIndicator(BuildContext context, Widget child, ScrollableDetails details) {
     // When modifying this function, consider modifying the implementation in
     // the base class as well.
-    final AndroidOverscrollIndicator indicator = _androidOverscrollIndicator
+    late final AndroidOverscrollIndicator indicator;
+    if (Theme.of(context).useMaterial3) {
+      indicator = AndroidOverscrollIndicator.stretch;
```

Comment:
This only applies to the Android platform, correct? We don't perform the overscroll on anything else?

Reply:
That is correct!